samples/Student Application Process/README.md

```diff
@@ -0,0 +1,195 @@
+﻿﻿
+# Student Application Process
+## Summary
+This application is built for prospective students at a university/college. It allows them to register and apply for a program at the college/university by submitting relevant details and documents. 
+
+By integrating Power Automate it becomes an end to end application where a student can submit an application and receive automatic emails regarding the status of their application. An admin approves the application and enters in test and interview scores. The student then receives the final email where they are either accepted or rejected into the university/college. You can find the entire application with the flows [here](https://drive.google.com/file/d/1DhIm47FtgJc2DvWGObuRVVE5kOLmsWuC/view?usp=sharing).
+
+Flow of the application:
+![picture of the sample](assets/flow.png)
+
+Login:
+![picture of the sample](assets/login.png)
+
+Register:
+![picture of the sample](assets/register.png)
```

Comment:
Can you please remove the company logo from this screen in the sample?

Reply:
@Laskewitz Im not entirely sure how to delete and add an image file to a pull request. Im a bit to new to this. could you help me out?

Reply:
Hi! Is it possible that I can make the required changes and submit a new pull request instead?

Happy holidays to you too!

Reply:
Hi @sam-404  - you can sure make the required changes and submit a new PR if you want to. Make sure to close this one first then. 👍🏼